File: C#/6kyu - E.cs
//Link -> https://www.codewars.com/kata/526571aae218b8ee490006f4

using System;
using System.Linq;

public class Kata
{
  public static int CountBits(int n)
  {
       var zero_one = Convert.ToString(Math.Abs(n), 2);
       int result = 0;

       result = zero_one.Count(i => i == '1');

       return result;
  }
}

File: php/6kyu - E.php
<?php

//Link -> https://www.codewars.com/kata/54bf1c2cd5b56cc47f0007a1

function duplicateCount($text) {
    $letters = str_split(strtolower($text));
    $count = array_count_values($letters);
    $duplicates = 0;

    foreach($count as $number){
       if($number > 1){
           $duplicates++;
       }
    }

    return $duplicates;
}

?>
